### DISCUSSION TITLE
HUD API calls for Challenge 1 income limits

### DISCUSSION THREAD

**Discussion by ryancaralis** [datakind/datakit-housing-fall-2024#17] [Show and tell]
Hi all,

I put together two functions in this file for getting the income limits at the county level for all counties in a state from the HUD API.

The API info and token access is here: https://www.huduser.gov/portal/datasets/il.html#query_2024

I was getting errors passing the county codes from the provided data to the call, so the first function gets the county codes from HUD for a specified state and then the second gets the income limit data for each county.

The file is in a zipped folder because .py files aren't supported as an attachment.

Hope this helps!

[HUD_API_Calls.zip](https://github.com/user-attachments/files/17051191/HUD_API_Calls.zip)

**Comment by jacobrharris** [comment]
@ryancaralis Great contribution. Thanks for sharing.
